test/e2e/auth/audit.go

```diff
@@ -630,7 +637,14 @@ var _ = SIGDescribe("Advanced Audit", func() {
 			expectedEvents = append(expectedEvents, t.events...)
 		}
 
-		expectAuditLines(f, expectedEvents)
+		err = wait.Poll(pollingInterval, pollingTimeout, func() (bool, error) {
```

Comment:
do we have any guarantees about the times?

Reply:
Added a comment